src/main/java/emu/grasscutter/game/activity/musicgame/MusicGameActivityHandler.java

```diff
@@ -1,91 +1,91 @@
-package emu.grasscutter.game.activity.musicgame;
-
-import emu.grasscutter.game.activity.ActivityHandler;
-import emu.grasscutter.game.activity.GameActivity;
-import emu.grasscutter.game.activity.PlayerActivityData;
-import emu.grasscutter.game.props.ActivityType;
-import emu.grasscutter.net.proto.ActivityInfoOuterClass;
-import emu.grasscutter.net.proto.MusicBriefInfoOuterClass;
-import emu.grasscutter.net.proto.MusicGameActivityDetailInfoOuterClass;
-import emu.grasscutter.utils.JsonUtils;
-
-import java.util.stream.Collectors;
-
-@GameActivity(ActivityType.NEW_ACTIVITY_MUSIC_GAME)
-public class MusicGameActivityHandler extends ActivityHandler {
-
-    @Override
-    public void onInitPlayerActivityData(PlayerActivityData playerActivityData) {
-        var musicGamePlayerData = MusicGamePlayerData.create();
-
-        playerActivityData.setDetail(musicGamePlayerData);
-    }
-
-    @Override
-    public void onProtoBuild(PlayerActivityData playerActivityData, ActivityInfoOuterClass.ActivityInfo.Builder activityInfo) {
-        MusicGamePlayerData musicGamePlayerData = getMusicGamePlayerData(playerActivityData);
-
-        activityInfo.setMusicGameInfo(MusicGameActivityDetailInfoOuterClass.MusicGameActivityDetailInfo.newBuilder()
-            .putAllMusicGameRecordMap(
-                musicGamePlayerData.getMusicGameRecord().values().stream()
-                    .collect(Collectors.toMap(MusicGamePlayerData.MusicGameRecord::getMusicId, MusicGamePlayerData.MusicGameRecord::toProto)))
-            .addAllPersonCustomBeatmap(musicGamePlayerData.getPersonalCustomBeatmapRecord().values().stream()
-                .map(MusicGamePlayerData.CustomBeatmapRecord::toPersonalBriefProto)
-                .map(MusicBriefInfoOuterClass.MusicBriefInfo.Builder::build)
-                .toList())
-
-            .addAllOthersCustomBeatmap(musicGamePlayerData.getOthersCustomBeatmapRecord().values().stream()
-                .map(MusicGamePlayerData.CustomBeatmapRecord::toOthersBriefProto)
-                .map(MusicBriefInfoOuterClass.MusicBriefInfo.Builder::build)
-                .toList())
-            .build());
```

Comment:
why？you have to make the builder extends ‘UgcMusicBriefInfoOuterClass’ instead of annotating them

Reply:
Now, of course, that's the way to go

Reply:
i have said musicgame was delete because of lack of protos.

Reply:
or add the old protos, but i think it makes no sense

Reply:
> or add the old protos, but i think it makes no sense

yes,I think so too